plugins/services/src/js/components/forms/GeneralServiceFormSection.js

```diff
@@ -321,6 +321,12 @@ class GeneralServiceFormSection extends Component {
     );
   }
 
+  moveCaretAtEnd(e) {
+    const temp_value = e.target.value;
+    e.target.value = "";
+    e.target.value = temp_value;
```

Comment:
Instead of mutating the input element's value how would you feel about using the inputs `setSelectionRange` method instead?

```
const length = e.target.value.length;
e.target.setSelectionRange(length, length);
```

Reply:
That's a good point, I will try it. Thank you.